.github/workflows/radar-snapshot-action.yml

```diff
@@ -0,0 +1,33 @@
+# Trigger an SDK publication to SNAPSHOTS repo on Maven Central when a commit is pushed to develop.
+name: Radar Snapshot Action
+on:
+  push:
+    branches:
+      - develop
+jobs:
+  Build-And-Publish:
+    runs-on: ubuntu-latest
+    steps:
+      # Prepare Environment
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Configure Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
+      # Gradle Build & Publish
+      - name: Build SDK
+        run: ./gradlew build
+      - name: Publish to Maven #and dispatch snapshot event
+        env:
+          NEXUS_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+          NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+          ORG_GRADLE_PROJECT_SIGNINGKEY: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEY }}
+          ORG_GRADLE_PROJECT_SIGNINGPASSWORD: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGPASSWORD }}
+#          PAT_USERNAME: ${{ secrets.PAT_USERNAME }}
+#          PAT_TOKEN: ${{ secrets.PAT_TOKEN }}
+          SNAPSHOT: true
```

Comment:
this seems like an exact copy of the release action but just with this param flipped - is there an easier way to specify this value without a completely new file? perhaps using `sed`?

Reply:
@jsani-radar my latest push adds some additional environment variables and gradle tasks. I can look into reusable containers within github actions as well, to see if that can help reduce some repeated configurations between these two scripts.

Reply:
There's a little bit on [composite steps](https://github.blog/changelog/2020-08-07-github-actions-composite-run-steps/), though not sure how useful it would be for now (there is not too much repetition)